cpp/refactor_module/algorithm/refactor.cpp

```diff
@@ -496,6 +496,8 @@ void NormalizeToBoolean(node_or_rel object, std::string &property_key, std::unor
     object.SetProperty(property_key, mgp::Value(false));
   } else if (!in_true && !in_false) {
     object.RemoveProperty(property_key);
+  } else {
+    throw mgp::ValueException("Property is contained in both true_values and false_values.");
```

Comment:
I think this is a more informative error message:
```suggestion
    throw mgp::ValueException(fmt::format("The value {} is contained in both true_values and false_values.", value.ToString()));
```

Reply:
Done without fmt as it felt unnecessary to add fmt to CMake for one use.